app/services/standardize_address_service.rb

```diff
@@ -4,6 +4,9 @@
 # PLEASE do NOT use this service to standardize addresses in bulk (do NOT use in backfills)
 # as that violates the terms of agreement with USPS and will result in access being revoked.
 class StandardizeAddressService
+  attr_accessor :timeout
+  alias_method :timeout?, :timeout
```

Comment:
ohh never seen this: is this so that you can call `timeout` and `timeout?`?

Reply:
yep!